components/SocialLoginGroup/SocialLoginGroup.css

```diff
@@ -94,3 +94,24 @@
   opacity: 0.8;
   cursor: pointer;
 }
+
+/* Github */
+.githubButton {
+  color: #24292e;
+  background: #eff3f6;
+  transition: opacity 0.15s linear;
+  border: 1px solid #1b1f23;
+}
+
+.githubButton::before {
+  border-right: #1b1f23 1px solid;
+  background: url("https://s3-us-west-2.amazonaws.com/s.cdpn.io/14082/icon_github.png") 6px 6px
```

Comment:
We have an SVG for the GitHub logo in `public/static/images/icons` - can you use that?

Reply:
this is what it looks like with the ones we have in `static/images/icons`

`github_logo_circle`: 
<img width="313" alt="image" src="https://user-images.githubusercontent.com/1694600/66937964-f3f60480-f005-11e9-80fe-c6228daf1b81.png">

`github_logo`
<img width="314" alt="image" src="https://user-images.githubusercontent.com/1694600/66938118-3ddeea80-f006-11e9-86a9-3cacf797700a.png">


Since it's a background-image I cannot really modify the colors of the svg (fill, color properties)

Let me know how do you want to proceed.

Reply:
I really don't mind that last one 🤷‍♂